test/bin/scenario.sh

```diff
@@ -1223,6 +1245,9 @@ if [ $# -lt 2 ]; then
     exit 1
 fi
 
+# Change added for USHIFT-5528
```

Comment:
When adding comment, it is often preferrable to explain in place rather than refer to external sources.